pontoon/checks/management/commands/run_checks.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+from celery import (
+    group,
+    signature,
+)
+from django.core.management.base import BaseCommand
+
+from pontoon.base.models import Translation
+from pontoon.checks import SUPPORTED_FILES
+
+import pontoon.checks.tasks
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Run checks on all translations'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--batch-size',
+            action='store',
+            dest='batch_size',
+            default=10000,
+            help='Number of translations to check in a single batch/Celery task',
+        )
+
+        parser.add_argument(
+            '--with-disabled-projects',
+            action='store_true',
+            dest='disabled_projects',
+            default=False,
+            help='Include disabled projects'
+        )
+
+        parser.add_argument(
+            '--with-obsolete-entities',
+            action='store_true',
+            dest='obsolete_entities',
+            default=False,
+            help='Include obsolete entities'
+        )
+
+    def handle(self, *args, **options):
+        filter_qs = {}
+
+        if not options['disabled_projects']:
+            filter_qs['entity__resource__project__disabled'] = False
+        else:
+            log.debug('Include disabled projects')
+
+        if not options['obsolete_entities']:
+            filter_qs['entity__obsolete'] = False
+        else:
+            log.debug('Include obsolete entities')
```

Comment:
Nit: I'd rather remove those debug messages, because they just print out what you explicitly request with parameters. Or you could add comments instead.